lms/templates/courseware/info.html

```diff
@@ -47,12 +47,10 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
   <div class="info-wrapper">
     % if user.is_authenticated():
     <section class="updates">
-      % if staff_access and masquerade is not UNDEFINED and studio_url is not None:
-        % if masquerade == 'staff':
-          <div class="wrap-instructor-info studio-view">
-            <a class="instructor-info-action" href="${studio_url}">${_("View Updates in Studio")}</a>
-          </div>
-        % endif
+      % if studio_url is not None and masquerade and masquerade.role == 'staff':
+        <div class="wrap-instructor-info studio-view">
+          <a class="instructor-info-action" href="${studio_url}">${_("View Updates in Studio")}</a>
```

Comment:
Currently, the `STUDIO_NAME` and `STUDIO_SHORT_NAME` constants only exist in`cms/envs/common.py` -- but they should be added to `lms/envs/common.py`, too. It would probably be better to move them to the `lms` env file and then import them into the `cms` one, same as we do with several other variables that are shared.

This change is not super important -- you're using the name "Studio" here, instead of "edX Studio", so it's fine. The reason for this change is to avoid using "edX" in our templates, which causes open source websites to violate edX's trademarks. Since this doesn't actually use the name "edX", it's fine to just merge the PR like this and clean it up afterward.